src/problem/constr/constr_NBOUND.c

```diff
@@ -598,8 +598,8 @@ void CONSTR_NBOUND_eval_step(Constr* c, Branch* br, int t, Vec* values) {
 	  BUS_has_flags(bus,FLAG_VARS,BUS_VAR_VMAG)) {
 
 	u = VEC_get(values,BUS_get_index_v_mag(bus,t));
-	umax = BUS_get_v_max(bus);
-	umin = BUS_get_v_min(bus);
+	umax = BUS_get_v_max_reg(bus);
+	umin = BUS_get_v_min_reg(bus);
```

Comment:
I agree it makes the most sense and that there are specific regularization functions and constraints that can be added specific to voltage regulating controls. I think it should be clear that voltage-magnitude-regularization should be for the ``v_max_norm`` and ``v_min_norm`` but it might be worth stating in the documentation along with "voltage * regulation by generator|shunts|tap-changers refer to ``v_max_reg`` and ``v_min_reg``.  What do you think?